rpcs3/Emu/Cell/Modules/sys_libc_.cpp

```diff
@@ -12,6 +12,10 @@ extern fs::file g_tty;
 
 // cfmt implementation (TODO)
 
+using qsortcmp = s32(vm::ptr<const void> e1, vm::ptr<const void> e2);
+
+static thread_local vm::ptr<qsortcmp> g_cmp;
```

Comment:
It better be declared inside the function as ``g_tls_cmp`` with type ``decltype(cmp)``